openstack/compute/v2/extensions/evacuate/requests.go

```diff
@@ -0,0 +1,41 @@
+package evacuate
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+// EvacuateOptsBuilder allows extensions to add additional parameters to the
+// the Evacuate request.
+type EvacuateOptsBuilder interface {
+	ToEvacuateMap() (map[string]interface{}, error)
+}
+
+// EvacuateOpts specifies Evacuate action parameters.
+type EvacuateOpts struct {
+	// The name of the host to which the server is evacuated
+	Host string `json:"host,omitempty"`
+
+	// Indicates whether server is on shared storage
+	OnSharedStorage bool `json:"OnSharedStorage"`
```

Comment:
The full line of this needs to be:

```go
OnSharedStorage bool `json:"onSharedStorage"`
```

Note how the Go field has a capital O but the JSON field is all lower-case.